### DISCUSSION TITLE
Is there a way to get a list of queue names?

### DISCUSSION THREAD

**Discussion by merterr** [rabbitmq/rabbitmq-server#11155] [Questions] [answered]
I want to create a web service to manage all queues. Every client  will use that service. They will pass queue name to that service. That service will create a queue with passed queue name.

**Comment by lukebakken** [comment]
https://www.google.com/search?q=get+list+of+queue+names+rabbitmq

**Reply by merterr** [reply]
thank you :) but I couldn't explain it fully. I want to use it via RabbitMQ.Client in .net core wep app.

**Reply by merterr** [reply, marked as answer]
I handled my problem using EasyNetQ.Management.Client packages.
